amp/WEB-INF/src/org/digijava/module/aim/util/AuditLoggerUtil.java

```diff
@@ -258,7 +258,63 @@ public static void logActivityUpdate(HttpServletRequest request, AmpActivityVers
             throw new RuntimeException(ex);
         }
     }
+    public static Collection<AmpAuditLogger> getFilteredUser(boolean withLogin, Long userid) {
```

Comment:
here i would use only one method and test if the params are null, then dont add them so we dont have so many methods

Reply:
yes that was what I was thinking